packages/ed25519-signature-2020/src/Ed25519KeyPair.test.ts

```diff
@@ -0,0 +1,114 @@
+import crypto from 'crypto';
+import { Ed25519KeyPair } from './Ed25519KeyPair';
+import * as fixtures from './__fixtures__';
+
+describe('generate', () => {
+  it('from random seed', async () => {
+    const key = await Ed25519KeyPair.generate({
+      secureRandom: () => {
+        return crypto.randomBytes(32);
+      },
+    });
+    expect(key.id).toBeDefined();
+    expect(key.type).toBe('Ed25519LinkedDataKeyPair2020');
```

Comment:
I agree, however I needed named types for the external and internal representation you can see my attempt to formalize the distinction here: https://or13.github.io/best-linked-data-suites-2020/#key-pair

I suppose a better approach would be to drop the year / date bits from the class and leave them on the external interface...  that way I can take your suggested change without creating an ambiguous type.

Reply:
Yeah +1 to this suggestion where the key pair is `Ed25519KeyPair2020 ` and the verification method is `Ed25519PublicKey2020` only because if we are using the term key in one we might as well use it in both.

Reply:
adopted.